src/components/organisms/PrivacyPreferenceCenter.module.css

```diff
@@ -105,6 +105,7 @@
   left: -25rem;
   opacity: 0;
   box-shadow: none;
+  pointer-events: none;
```

Comment:
I needed to change this because it was very hard to click on the previous step button on the laptop screen, but you are right, should I open a separate PR then?

Reply:
Its fine for now from my pov - just keep it in mind for further PRs